src/pages/Profile/index.tsx

```diff
@@ -219,10 +221,13 @@ const mapStateToProps: MapStateToProps<StateProps, OwnProps, AppState> = (
   ownProps
 ) => {
   const loggedInMember = getLoggedInMember(state);
-  const profileMember = state.membersNew.byMid[ownProps.match.params.memberMid];
+  const memberUsername = ownProps.match.params.memberUsername;
+  const profileMember = state.membersNew.byUsername[memberUsername];
   if (!profileMember) {
     // trust action could not have been initiated if profile never was initialized
-    return { loggedInMember };
+    // tslint:disable-next-line:no-debugger
+    const isLoading = Object.keys(state.membersNew.byUsername).length === 0;
+    return { isLoading, loggedInMember, memberUsername };
```

Comment:
@osdiab agree that this is wrong but I ran into some trouble merging memberUsername into props, feel free to edit after land